Deploy/kubernetes/pv/log.pv.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: log-pv
+  labels:
+    type: log
+spec:
+  capacity:
+    storage: 5Gi
```

Comment:
How do you know 5Gi is enough? This is a magic number which we don't really want to use in production.